samples/sample-bugsnag/settings.gradle.kts

```diff
@@ -12,6 +12,14 @@ include(":app", ":shared")
 
 enableFeaturePreview("VERSION_CATALOGS")
```

Comment:
If you update the version of gradle, you can drop this feature preview in all the files as _version catalogs_ are now a stable gradle feature.